### DISCUSSION TITLE
<title>BIND "unable to create service account" error

### DISCUSSION THREAD

**Discussion by serhandonertas** [community/community#46049] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone,
I can't install BIND. "Unable to create service account" error. What should I do? How can I install BIND?
Thank you.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by serhandonertas** [reply]
thanks for your reply. 
I've tried custom domain website but this doesn't work because I didn't download and apply BIND. And I've realized that I need windows server for applying BIND. Windows server wants me to use Azure for server. Finally I give up for publishing custom domain web page.
Thanks again.
